test/e2e/customer_admin_cluster_test.go

```diff
@@ -0,0 +1,82 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure customer-cluster-admin e2e tests [CustomerAdmin]", func() {
+	defer GinkgoRecover()
+
+	It("should not read nodes", func() {
+		_, err := cadmin.kc.CoreV1().Nodes().Get("master-000000", metav1.GetOptions{})
+		Expect(kerrors.IsForbidden(err)).To(Equal(true))
+	})
+
+	It("should have full access on all non-infrastructure namespaces", func() {
+		// Create project as normal user
+		namespace := nameGen.generate("e2e-test-")
+		c.createProject(namespace)
+
+		err := wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			rb, err := cadmin.kc.RbacV1().RoleBindings(c.namespace).Get("customer-admin", metav1.GetOptions{})
+			if err != nil {
+				return false, nil
+			}
+			for _, subject := range rb.Subjects {
+				if subject.Kind == "Group" && subject.Name == "customer-admins" {
+					return true, nil
+				}
+			}
+			return false, nil
```

Comment:
This should be `false, errors.New("customer-admins rolebinding does not bind to customer-admins group")`.

Reply:
Thanks, will fix.